db/config.cc

```diff
@@ -1023,6 +1027,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , auth_certificate_role_queries(this, "auth_certificate_role_queries", value_status::Used, { { { "source", "SUBJECT" }, {"query", "CN=([^,]+)" } } },
         "Regular expression used by CertificateAuthenticator to extract role name from an accepted transport authentication certificate subject info.")
     , error_injections_at_startup(this, "error_injections_at_startup", error_injection_value_status, {}, "List of error injections that should be enabled on startup.")
+    , enable_maintenance_port(this, "enable_maintenance_port", value_status::Used, false, "If set to true, Scylla will listen on a maintenance port on localhost. WARNING: This port is not secured and should not be exposed to the internet.")
```

Comment:
Why not default to true?

Reply:
Now it opens a port available for everyone on the machine. It can be very risky if admins are not aware of this.

Reply:
Haven't you agreed to remove this ansd use the unix socket instead?

Reply:
It seems you don't need two separate config variables. If --maintenance-mode=/path/to/unix-socket is enabled, the mode is enabled, otherwise not, but either way, why port?